open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def drop_columns(df,columns):
</pre><p>Need a space after <code>,</code></p><p> </p><p>You can actually format the code https://mkyong.com/vscode/how-to-format-source-code-in-visual-studio-code-vscode/</p><p> </p><p>If needed, you can install the black plugin in the vscode. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>DONE</p><p> </p>